sdk/trace/batch_span_processor.go

```diff
@@ -84,6 +83,7 @@ type batchSpanProcessor struct {
 	stopWait   sync.WaitGroup
 	stopOnce   sync.Once
 	stopCh     chan struct{}
+	stopped    atomic.Bool
```

Comment:
in what way is an atomic.Bool different from a chan WRT signaling shutdown?  If we need the atomic.Bool, can we replace usage of stopCh with this new field?

Reply:
It primarily used mostly used to handle `OnEnd` after `Shutdown` is called to make sure that spans are not further enqueued.